package.json

```diff
@@ -231,6 +231,22 @@
           "markdownDescription": "Insert space before end of self closing tag. \nExample:\n  ```<tag/> -> <tag />```. Default is `true`. See [here](https://github.com/redhat-developer/vscode-xml/wiki/Formatting#xmlformatspacebeforeemptyclosetag-) for more information.",
           "scope": "window"
         },
+        "xml.format.xsiSchemaLocationSplit": {
+          "type": "string",
+          "enum": [
+            "onNamespace",
+            "onLocation",
+            "none"
```

Comment:
Update to `"onElement"`, `"onPair"`, `"none"`;

Reply:
fixed